client/src/app.js

```diff
@@ -0,0 +1,144 @@
+import io from "socket.io-client";
+
+const ENDPOINT = "http://www.toonin.ml:8100/";
+
+const servers = {
+    iceServers: [
+      {
+        urls: [
+          "stun:stun.l.google.com:19302",
+          "stun:stun2.l.google.com:19302",
+          "stun:stun3.l.google.com:19302",
+          "stun:stun4.l.google.com:19302"
+        ]
+      }
+    ]
+};
+
+var socket;
+var audioElem;
+
+export function init(audioElement) {
+    audioElem = audioElement;
+    setSocketListeners = setSocketListeners.bind(this);
+    createAnswer = createAnswer.bind(this);
+}
+
+export function toonin() {
+    return;
+}
+
+export function logMessage(msg) { console.log(msg); }
+
+export function checkstream() {
+    var obj = {
```

Comment:
Could be renames to state and made a const variable

Reply:
In fact, we may want to refactor this state to Vuex so that we could keep track of where we are in the connection process and update it on the UI